DPGAnalysis/Phase2L1TNanoAOD/python/l1tPh2Nanotables_cff.py

```diff
@@ -24,6 +24,29 @@
      )
  )
 
+#### GTT Displaced Vertex
+dispVtxTable = cms.EDProducer(
+    "SimpleL1DisplacedVtxCandidateFlatTableProducer",
+    src = cms.InputTag('DisplacedVertexProducer','dispVertices'),
+    cut = cms.string(""),
+    name = cms.string("L1DispVertex"),
+    doc = cms.string("GTT Displaced Vertices"),
+    singleton = cms.bool(False), # the number of entries is variable
+    variables = cms.PSet(
+        d_T = Var("d_T()",float, doc = "impact parameter of parent particle"),
+        R_T = Var("R_T()",float, doc = "transverse distance of vertex from origin"),
+        cos_T = Var("cos_T()", float, doc = "cosine of angle between parent particle momentum and position vector of vertex"),
+        x = Var("x()", float, doc = "x position of vertex"),
+        y = Var("y()", float, doc = "y position of vertex"),
+        z = Var("z()", float, doc = "z position of vertex"),
+        openingAngle = Var("openingAngle()", float, doc = "hardware track multiplicity in the vertex"),
```

Comment:
Is this description (`doc="..."`) correct? (Sorry my ignorance ...)

Reply:
Thanks for the catch! I had copy pasted from https://github.com/cms-sw/cmssw/blob/7bc980b91bfda320c31a55ec6e943a17772cb95d/DPGAnalysis/Phase2L1TNanoAOD/python/l1tPh2Nanotables_cff.py#L20 and forgotten to update the documentation for that variable. Now fixed with https://github.com/cms-sw/cmssw/pull/48326/commits/310ef66767317b6f091c9c7fd865f4d4838a8945